Dockerfile

```diff
@@ -12,11 +12,11 @@ RUN apt-get update -qqy && DEBIAN_FRONTEND=noninteractive apt-get install -qqy -
     unzip \
 && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
 
-ARG VERSION=4.8.5
-ARG FILE=z3-$VERSION-x64-debian-8.11
+ARG VERSION=4.8.8
+ARG FILE=z3-$VERSION-x64-ubuntu-16.04
```

Comment:
The base image uses actually Debian Buster (10), which is much newer than Debian 8 or Ubuntu 16, but I guess it works anyway. Python Z3 seems to have a PyPi package too, so installing with `pip install` should be much simpler than the old way of copying files, but there is no need to change this if it still works.